.env.example

```
@@ -33,6 +33,10 @@ VITE_APP_TITLE=Code Review Admin
 VITE_APP_DESCRIPTION=AI代码审查管理系统
 VITE_DEV_PROXY_TARGET=http://backend:8000
 
+# 前端登录配置
+VITE_DEFAULT_USERNAME=admin
+VITE_DEFAULT_PASSWORD=admin123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_DEFAULT_PASSWORD key should go before the VITE_DEFAULT_USERNAME key

(UnorderedKey)